src/Builder/FormContractor.php

```diff
@@ -98,6 +98,9 @@ public function getFormBuilder($name, array $options = [])
 
     public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescription)
     {
+        // NEXT_MAJOR: Remove this line and update the function signature.
+        $formOptions = \func_get_args()[2] ?? [];
```

Comment:
Can you add phpdoc for the new parameter? Maybe use `@phpstan-param` if the php-cs-fixer removes this.

Reply:
@core23 The phpdoc is inherited from the interface getDefaultOptions.

I prefer to add the phpdoc in the interface in the SonataAdmin PR https://github.com/sonata-project/SonataAdminBundle/pull/6438/files